task/dev3/EnterUser/EnterUser.java

```diff
@@ -0,0 +1,29 @@
+package task.dev3.EnterUser;
+
+import task.dev3.User.*;
+
+import java.io.*;
+import java.util.*;
+
+/**
+ * Class set entered information.
+ */
+public class EnterUser {
+
+  /**
+   * Call other method for giv information about user and set it.
+   * @param user - user, which need to initialize.
+   */  
+  public void setEnterUserInfo(User user) {
```

Comment:
set supposes it setter i.e. it changes the internal fields of EnterUser instance. Better just return User and anme method like readUser